platform/core/src/types/DisplaySet.ts

```diff
@@ -13,6 +13,9 @@ export type DisplaySet = {
   Modality?: string;
   imageIds?: string[];
   images?: unknown[];
+  label?: string;
+  /** Flag indicating if this is an overlay display set (e.g., SEG, RTSTRUCT) */
```

Comment:
This is very useful to have declared since the DICOM standard way to store non-RT contours is with an SR like we do measurements.